user-guide/network/nictag.rst

```diff
@@ -46,13 +62,53 @@ Adding a NIC Tag
         external       -                  aggr0
         admin          -                  aggr0
 
+Adding etherstub NIC tag
+------------------------
+
+Etherstubs are created with the command shown below. Please make sure that etherstub name ends with a number, otherwise you will end up with an error.
+Etherstubs created this way will be written to ``/usbkey/config``, and this makes them persistent over the reboots.
+
+.. code-block:: bash
+
+        [root@node01 ~] nictagadm add -l stub0
+        [root@node01 ~] nictagadm list
+        NAME           MACADDRESS         LINK           TYPE
+        stub0          -                  -              etherstub
+
+Adding overlay NIC tag
+----------------------
+
+As opposed to other NIC tag types, overlays are not created using ``nictagadm`` command. They are created by adding ``/var/run/smartdc/networking/overlay_rules.json`` and ``/var/run/smartdc/networking/my_overlay1.json`` files.
+
+``overlay_rules.json`` provides information to the operating system about properties of the overlays. Here is the example of how this file might look like:
+
+.. code-block:: bash
+
+        [root@node01 ~] cat /var/run/smartdc/networking/overlay_rules.json
+        {
+        "my_overlay1": "-e vxlan -p vxlan/listen_ip=192.168.100.100,vxlan/listen_port=4790 -s files -p files/config=/var/run/smartdc/networking/my_overlay1.json -p mtu=1400",
+        "my_overlay2": "-e vxlan -p vxlan/listen_ip=192.168.200.200,vxlan/listen_port=4791 -s files -p files/config=/var/run/smartdc/networking/my_overlay2.json -p mtu=1400",
+        "my_overlay3": "-e vxlan -p vxlan/listen_ip=0.0.0.0,vxlan/listen_port=4790 -s files -p files/config=/var/run/smartdc/networking/my_overlay3.json -p mtu=1400"
+        }
+
 Deleting a NIC Tag
 ------------------
 
+The ``nictagadm delete`` command should be used to delete NIC tags.
+
 .. code-block:: bash
 
     [root@node01 ~] nictagadm delete external
     [root@node01 ~] nictagadm list
     NAME           MACADDRESS         LINK           TYPE
     admin          78:24:af:9c:3b:53  rge0           normal
 
+Overlays can be deleted by using both ``dladm`` or ``nictagadm``, though ``nictagadm`` is prefered due to the consistency.
```

Comment:
which consistency? :) ~~the~~

Reply:
Consistency with everything written in the current section, thanks for reading document as whole :) 
in the Managing NIC Tags there is a warning, that dladm should not be used with objects that were created using nictagadm.

However, I'll update this, because overlays can not be deleted using nictagadm, since command doesn't support deletion of overlays, or rather returns 0, like it succeeded, but overlay stays live and well.